CONTRIBUTION

```diff
@@ -1,25 +1,24 @@
 Making Contributions
 
-Cothority is an Open Source program with many contributors (listed in the AUTHORS
-files), so we actively seek contributions (help on the Cothority users list,
-documentation, source code, ideas, …).  All contributions help us make a
-better product.
+Onet is an Open Source program with many contributors (listed in the AUTHORS
+files), so we actively seek contributions (documentation, source code, ideas, …).
+All contributions help us make a better product.
```

Comment:
`product` vs `Open Source program` ?
It sounds contradictory, maybe`better software` is a better fit or stg else. 
Also, `open source` does not have to be capitalized, I guess ?